packages/node-runtime-worker-thread/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "@mongosh/node-runtime-worker-thread",
+  "version": "0.6.1",
+  "description": "MongoDB shell runtime that lives in a worker thread",
+  "homepage": "https://github.com/mongodb-js/mongosh",
+  "license": "Apache-2.0",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/mongosh"
+  },
+  "engines": {
+    "node": "^12.4.0"
```

```suggestion
    "node": ">=12.4.0"
```

(Is this copied from the `types` package or set up this way by lerna directly? I don’t know how that happened there either, it looks a bit like lerna did that by default, which is … weird – I’ll be pushing a fix for the `types` package to master directly since it’s super trivial)

---

Ooooh, it was copied, not from `types` though, but from `@mongosh/autocomplete`, so you might want to add it also to your fix. Thanks for spotting!